src/assets/styles/base/_base.scss

```diff
@@ -35,7 +35,84 @@ html {
   --color-paragraph: #{$color-coal};
   --color-share-icons: #{$color-majorelle};
   --color-social-icons: #{$color-cerulean};
+  --color-close-bg: #{$color-snow};
   --color-close: #{$color-cerulean};
+
+  @media screen and (min-width: map-get($breakpoints, 'breakpoint-11')) {
+    --color-close-bg: #{$color-ivory};
+  }
+}
+
+.page--holiday-extras {
+  --color-accent: #{$holiday-extras-gold};
+  --color-background: #{$holiday-extras-purple-3};
```

Comment:
I'm not a fan of the number system, as it means nothing to me. Would prefer either a colour naming (e.g. holiday-extras-violet) or scale (e.g. holiday-extras-purple-lighter) than numbers. I'd be happy to use `hx-` rather than `holiday-extras-` too.